src/app/services/init.service.ts

```diff
@@ -218,7 +218,9 @@ export class InitService {
     //   .catch(() => {
     //     // throw new DataResponseError('COOKIE_SET_FAILURE')
     //   })
-    this.logFirstLogin()
+    if (!window.location.href.includes('/public/')) {
```

Comment:
It should only for non public users?

Reply:
If the user accessing the page means he is non loggedin.